test/createAzureResource.test.ts

```diff
@@ -53,23 +53,6 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         ext.tree.dispose();
     });
 
-    test('Create and Delete New Function App', async () => {
-        const resourceName: string = getRandomHexString().toLowerCase(); // storage accounts cannot contain upper case chars
-        resourceGroupsToDelete.push(resourceName);
-
-        const testInputs: string[] = [resourceName, '$(plus) Create new resource group', resourceName, '$(plus) Create new storage account', resourceName, 'West US'];
-        ext.ui = new TestUserInput(testInputs);
-        await vscode.commands.executeCommand('azureFunctions.createFunctionApp');
-        const client: WebSiteManagementClient = getWebsiteManagementClient(testAccount);
-        const createdApp: WebSiteManagementModels.Site = await client.webApps.get(resourceName, resourceName);
-        assert.ok(createdApp);
-
-        ext.ui = new TestUserInput([resourceName, DialogResponses.deleteResponse.title, DialogResponses.yes.title]);
```

Comment:
Need to get rid of the reference to DialogResponse at the top.  It causes a build error.